daemon/mgr/container.go

```diff
@@ -695,7 +696,7 @@ func (mgr *ContainerManager) Stop(ctx context.Context, name string, timeout int6
 	c.Lock()
 	defer c.Unlock()
 
-	if !c.IsRunning() {
+	if !c.IsRunning() && !c.IsPaused() {
```

Comment:
maybe we should also check `Restart` interface

Reply:
done it